.travis.yml

```diff
@@ -4,6 +4,8 @@ script:
   - cargo build --verbose
   - cargo test --verbose
   - cargo doc
+  - cd regex_macros
+  - cargo test --verbose
```

Comment:
This could also be replaced with `cargo test --manifest-path=regex_macros/Cargo.toml` (avoiding the `cd`)